weneige/products/models.py

```diff
@@ -0,0 +1,85 @@
+from django.db import models
+
+class Product(models.Model):
+    kor_name    = models.CharField(max_length=100)
+    eng_name    = models.CharField(max_length=100)
+    price       = models.DecimalField(max_digits=7, decimal_places=0)
+    description = models.TextField(max_length=100)
+
+    class Meta:
+        db_table = "products"
+
+class Tag(models.Model):
+    product = models.ManyToManyField("Product", through="ProductTag", related_name="tag")
+    name    = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "tags"
+
+class ProductTag(models.Model):
+    product = models.ForeignKey("Product", on_delete=models.CASCADE)
+    tags    = models.ForeignKey("Tag", on_delete=models.CASCADE)
```

Comment:
```suggestion
    tag    = models.ForeignKey("Tag", on_delete=models.CASCADE)
```

복수 단수도 가독성 있는 코드를 위해 컨벤션에 유의해주세요~